backend/conftest.py

```diff
@@ -230,6 +231,12 @@ async def send_task_mock() -> MagicMock:
     return MagicMock(spec=send_task)
 
 
+@pytest.fixture
+def smtplib_mock() -> MagicMock:
+    with patch("smtplib.SMTP", autospec=True) as mock:
```

Comment:
I went with patch here because I couldn't get the MagicMock (used elsewhere in these tests) to work. I've never been great at mocking, so I'm sure there's a way to do it properly.

Reply:
It didn't work :( The tests all failed with a `ConnectionRefusedError: [Errno 61] Connection refused` error.

Reply:
Yeah, actually that was pretty dumb on my part: it couldn't work since we do need to patch the standard library; it can't just be a variable we're passing around. Apologies 😊